thrift/src/main/thrift/atoms/recipe.thrift

```diff
@@ -32,25 +31,33 @@ struct Range {
 struct Ingredient {
   1: required string item
   2: optional string comment
-  3: optional double quantity
-  4: optional Range quantityRange
-  5: optional string unit
+  3: required Range quantityRange
```

Comment:
Similarly to removing fields, adding new ones would involve commenting out the existing and starting the new numbering where the old left off, e.g. `6: required Range quantityRange` etc. And let's make absolutely certain when Required is needed because adding we would need to reindex in order to backfill any old required values. Wherever possible making new fields `optional` is much more flexible.